From 5ed33f830410f37506e6cf0242ec86d1a56448e6 Mon Sep 17 00:00:00 2001
From: Ying Xie <ying.xie@microsoft.com>
Date: Wed, 25 Nov 2020 22:28:43 +0000
Subject: [PATCH] frr: remove duplicate file log outchannel

SONiC runs FRR inside a container and forwards daemon logs through rsyslog.
Remove the duplicate /var/log/frr/frr.log outchannel to avoid unbounded
container-local log growth.

Original-patch-commit: 0417036a3aece862c95111d4646ca7508a028165
---
 tools/etc/rsyslog.d/45-frr.conf | 24 ------------------------
 1 file changed, 24 deletions(-)

diff --git a/tools/etc/rsyslog.d/45-frr.conf b/tools/etc/rsyslog.d/45-frr.conf
index ef37d66d82..2e8e1e12d1 100644
--- a/tools/etc/rsyslog.d/45-frr.conf
+++ b/tools/etc/rsyslog.d/45-frr.conf
@@ -2,30 +2,6 @@
 # to /var/log/frr/frr.log, then drops the message so it does
 # not also go to /var/log/syslog, so the messages are not duplicated
 
-$outchannel frr_log,/var/log/frr/frr.log
-if  $programname == 'babeld' or
-    $programname == 'bgpd' or
-    $programname == 'bfdd' or
-    $programname == 'eigrpd' or
-    $programname == 'frr' or
-    $programname == 'isisd' or
-    $programname == 'fabricd' or
-    $programname == 'ldpd' or
-    $programname == 'mgmtd' or
-    $programname == 'nhrpd' or
-    $programname == 'ospf6d' or
-    $programname == 'ospfd' or
-    $programname == 'pimd' or
-    $programname == 'pim6d' or
-    $programname == 'pathd' or
-    $programname == 'pbrd' or
-    $programname == 'ripd' or
-    $programname == 'ripngd' or
-    $programname == 'vrrpd' or
-    $programname == 'watchfrr' or
-    $programname == 'zebra'
-    then :omfile:$frr_log
-
 if  $programname == 'babeld' or
     $programname == 'bgpd' or
     $programname == 'bfdd' or
-- 
2.34.1

